src/utils.jl

```diff
@@ -0,0 +1,50 @@
+function axis_types(wcs::WCSTransform)
+    result = []
```

Comment:
This function needs a redesign.  Initialising an array in this way in Julia is usually a bad idea: to write performant code in Julia you need to make variables type stable, but here `result` is a `Vector{Any}`.  You actually know that the output will be a vector of two strings.  A better initial value could be `["", ""]` and use `result[axis]` instead of `subresult`.  Please, have a look at the [performance tips](https://docs.julialang.org/en/v1/manual/performance-tips/).

However, I am pretty sure that [broadcast](https://docs.julialang.org/en/v1/manual/functions/#man-vectorized-1) would make this function *way* simpler and faster.  Just write a function for one axis (for example it could be `axis_type`?), and this would simply become
```julia
axis_types(wcs::WCSTransform) = axis_type.(wcs.ctype)
```